src/models/data.h

```diff
@@ -69,6 +68,13 @@ struct Symbol
     {
         return !symbol.isEmpty() && !path.isEmpty() && relAddr > 0 && size > 0;
     }
+
+    bool isBinarySymbol() const
+    {
+        // a "binary only" symbol like used in "Top HotSpots Per File" has these characteristics
```

Comment:
that's probably something we should change/fix on another level rather than hacking it in here

Reply:
probably, but I'd need directions or someone else to pick that up

Reply:
I don't have the time for that so let's ignore this for now and at some point I'll pick it up again I hope :)